products-built-on-oxen/lokinet/guides/lokinet-on-a-soho-router.md

```diff
@@ -0,0 +1,325 @@
+# running lokinet client on a soho router
+
+the nicest way to run lokinet is on a soho router, it will make the defaults on the local network it serves use lokinet dns and even tunnel all traffic over lokinet in a way that it cannot leak.
+
+## requirements
+
+hardware with:
+
+* 1 wired interface 
+* 1 wireless interface
```

Comment:
or a second wired interface

Reply:
this guide includes setting up hostapd. so it is specifically for 1 wired and one wireless.